### ISSUE TITLE
Suggestion: add option to disable mouse acceleration in mouse settings

### ISSUE THREAD

**Issue by catman1900** [budgie-desktop/budgie-desktop#1120]
title

**Comment by Frampis** [comment]
This would be an extremely useful feature for me as well. It's especially important for gaming. It would be great if there was an easy way in the GUI to achieve true 1:1 mouse settings.

In Windows the equivalent is to set mouse sensitivity to 6/11 notches and turning "enhance pointer precision" off.

**Comment by YouveGoatToBeKittenMe** [comment]
Would love to have this as well

**Comment by Z3r0way** [comment]
You can do it by installing the gnome tweak tool and by setting the mouse acceleration profile to flat :)

**Comment by 3vilgenius** [comment]
Does that tool work for the budgie desktop environment? @Z3r0way

**Comment by Z3r0way** [comment]
@3vilgenius Yes because budgie is based on gnome. So you can use the gnome tweak tool 😉

**Comment by JoshStrobl** [comment]
@Z3r0way Budgie leverages specific GNOME technologies like Mutter, GSD, and GTK+. It isn't "based" on GNOME.
